playbooks/uda-workflow/connector-install/install_sap_hana_plugin.yml

```diff
@@ -0,0 +1,68 @@
+# => Cohesity Agent Management
+# =>
+# => Collection: cohesity.dataprotect
+# =>
+
+# => Install the Cohesity Datastore Agent on each SapHana host
+# => specified in the Ansible inventory
+# =>
+---
+- hosts: saphana
+  # => We need to gather facts to determine the OS type of
+  # => the machine
+  gather_facts: true
+  vars:
+    state: "present"
+    platform: SapHana
+    saphana_admin: sssadm
+    saphana_password: fr8shst8rt
+    saphana_home: /usr/sap/SSS/home
+    SID: SSS
+    user_store_key: SYS_KEY
+  collections:
+    - cohesity.dataprotect
+  tasks:
+    - name: Check if firewall is enabled on CentOS or RedHat
```

Comment:
If firewall is disabled we should ignore the next cmd , to open 11113  ? Can you make it conditional

Reply:
We do check the firewall status before allowing the ports.